Plugins/AssetUsageDetector/Editor/ObjectToSearch.cs

```diff
@@ -95,6 +95,21 @@ private void AddSubAssets( Object target, bool includeTarget, bool? shouldSearch
 					}
 				}
 
+				// If is an Atlas we get sprites directly as GetSubAssets return the final texture
+				if ( target is UnityEngine.U2D.SpriteAtlas spriteAtlas )
```

Comment:
- Can you add a `using` statement for SpriteAtlas instead?
- Since the new code's style is almost the same as the overall coding style of the plugin, I'll ask you to remove the space between `if` and the opening parantheses to match the style exactly.